runtime/js/40_http.js

```diff
@@ -14,6 +14,8 @@
     return new HttpConn(rid);
   }
 
+  const connErrors = new WeakMap();
```

Comment:
Do you have any reason to use WeakMap over a simple property like httpConn.error (`Error | undefined`)?

Reply:
To properly hide it from the public API.

Reply:
Then use a private property or symbol property.

Reply:
We use symbols for essentially all web APIs now. Should be fine.

Reply:
Replaced with a symbol.